### ISSUE TITLE
Escape backslashes when generating assets object for kotlin js

### ISSUE THREAD

**Issue by wakaztahir** [icerockdev/moko-resources#423]
error when compiling on windows , I know you guys are going to take like an year to fix this.

```
Illegal escape: '\e'
```

in generated assets object which contains

```
  internal actual object assets : ResourceContainer<AssetResource> {
    public actual val export_from_sketchable: AssetResource = AssetResource(
            originalPath = js("require(\"files\export-from-sketchable.md\")") as String, 
            rawPath = "export-from-sketchable.md"
        )
}
```

**Comment by wakaztahir** [comment]
tbh this library has made my brain go insane because of such low support on windows platform

**Comment by Alex009** [comment]
will be released in 0.21.0
